upgrade.md

```diff
@@ -1,9 +1,9 @@
 # Firmware Upgrade
 
-There are many ways to update your Black Magic Probe. For most users the stable release firmware will be sufficient and
-in most cases will not require to compile anything. We recommend updating the firmware using `bmputil`. It is also
-possible to use `dfu-util` that used to be the recommended way in the past, you can find the instructions for *other*
-tools at the end of this document. If you have trouble using `bmputil` please let us know, we strive to make the
+There are many ways to update your Black Magic Probe. For most users, the stable release firmware will be sufficient and
+in most cases will not require to compile anything. We recommend updating the firmware using `bmputil`, however if
+you want to use a different tool (`dfu-util`, `stlink-tool`, etc) you can find some instructions for *other*
```

Comment:
This needs to be careful not to imply that `stlink-tool` can be used to upgrade BMPs - it can only be used to upgrade ST-Link v2's running the BMD firmware. For why we're being picky about this, it's part of trying to enforce the project's [terminology](https://black-magic.org/knowledge/terminology.html) to help keep things disambiguated.